crates/iota/src/validator_commands.rs

```diff
@@ -685,9 +691,11 @@ impl IotaValidatorCommand {
                     .governance_api()
                     .get_latest_iota_system_state()
                     .await?;
-                let active_validators = match iota_system_state {
-                    IotaSystemStateSummary::V1(v1) => v1.active_validators,
-                    IotaSystemStateSummary::V2(v2) => v2.active_validators,
+                let (active_validators, committee_members) = match iota_system_state {
+                    IotaSystemStateSummary::V1(v1) => (v1.active_validators, None),
```

Comment:
in case the protocol is using v1 then all `active_validators` are `committee_members`